types/entry.go

```diff
@@ -0,0 +1,7 @@
+package types
+
+type Foo struct {
```

Comment:
There may be multiple types in the future. I wouldn't like to rename a package after adding a new type.